robottelo/config/base.py

```diff
@@ -178,6 +178,10 @@ def get_credentials(self):
         """
         return (self.admin_username, self.admin_password)
 
+    def get_hostname(self, key="hostname"):
+        reader = INIReader(os.path.join(get_project_root(), SETTINGS_FILE_NAME))
+        return reader.get('server', key, self.hostname)
```

Comment:
why using INIReader again (re-reading ini) if you have `settings.configure()`ed already? 
```
def get_hostname(self, key="hostname"):
    return settings.server.__dict__[key] or settings.server.hostname
```

Reply:
that's an option, but I try to avoid using dunder attributes within robottelo

Reply:
You could use `getattr`, then:
```python
def get_hostname(self, key="hostname"):
    return getattr(self.server, key, settings.server.hostname)
```

As discussed in #8003 , settings module is not ideal place for helpers like that. I would like to avoid introducing new one only to have it yanked in few months.

Reply:
@mirekdlugosz So this won't work, since the worker id key-value pairs aren't read into the server class. Adding the helper method is the cleanest way to handle this until we can revisit after dynaconf integration is complete.

Reply:
time being hack, OK   yet how many times yet :-)